src/Symfony/Bundle/WebServerBundle/Command/ServerStatusCommand.php

```diff
@@ -47,10 +48,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output);
         $server = new WebServer();
-        if ($server->isRunning($input->getOption('pidfile'))) {
-            $io->success(sprintf('Web server still listening on http://%s', $server->getAddress($input->getOption('pidfile'))));
+        if ($filter = $input->getOption('filter')) {
+            if ($server->isRunning($input->getOption('pidfile'))) {
+                list($host, $port) = explode(':', $address = $server->getAddress($input->getOption('pidfile')));
+                if ('address' === $filter) {
+                    $output->write($address);
```

Comment:
input comes from the pidfile. If you mess up with it, then you are on your own anyway.